astropy/table/tests/test_showtable.py

```diff
@@ -40,9 +41,11 @@ def test_stats(capsys):
                     ' name     mean      std    min  max {0}'
                     '------ --------- --------- ---- ----{0}'
                     'target        --        --   --   --{0}'
-                    ' V_mag 12.866668 1.7211105 11.1 15.2{0}')
+                    ' V_mag 12.86666? 1.7211105 11.1 15.2{0}')
 
-    assert out == expected.format(os.linesep)
+    # Here we use fnmatch to allow the use of wildcards (specifically ?) in the
+    # about output for cases where the values shown are platform-dependent.
+    assert fnmatch(out, expected.format(os.linesep))
```

Comment:
Just a minor remark, `fnmatch` is specialized for filenames so it can be a little confusing to use it here. Why not using `re.match` directly ?

Reply:
Oh and btw this reminds me that I converted all the other tests to use `.splitlines()` instead of having to handle the end-of-line character, I should do that here as well.

Reply:
@saimn - I've changed to ``re.match`` - feel free to open a PR to use splitlines